docs/mine/lotus/miner-wallets.md

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Lotus Miner: Wallets'
+description: 'Miners make use of wallets managed by the Lotus Node to both send and receive FIL during the normal course of operation (sending proofs, collecting deal payments etc.). This guides explains how to configure different wallets so the miner can use them for different operations.'
+breadcrumb: 'Miner Wallets'
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }}
+
+As mentioned during the [init phase](miner-setup.md) of the Lotus Miner, it is required to provide an owner address with some funds that the miner will use for all the operations needed (or it will use the [Lotus Node default wallet](../../get-started/lotus/send-and-receive-fil.md)). This address will be used by the miner for all the operations, but in many cases, it makes sense to use dedicated wallets:
+
+However, different components of the miners can be configured to use different wallets:
+
+[[TOC]]
+
+## The owner address
+
+The owner address is used to receive payments and to pay the fees for any miner-actor triggered changes like changing its information on changing addresses (TODO: actually list exactly what it is used for).
```

Comment:
I would like to have a list of things this wallet is actually used for if there is a separate worker wallet defined.